src/main/kotlin/com/ampnet/projectservice/controller/OrganizationInvitationController.kt

```diff
@@ -114,16 +114,15 @@ class OrganizationInvitationController(
     }
 
     private fun validateEmails(emails: List<String>) {
-        val regex = "^[a-zA-Z0-9_+&*-]+(?:\\.[a-zA-Z0-9_+&*-]+)*@(?:[a-zA-Z0-9-]+\\.)+[a-zA-Z]{2,7}$"
-        val pattern = Pattern.compile(regex)
-        val errorEmails = mutableListOf<String>()
-        emails.forEach {
-            if (!pattern.matcher(it).matches()) errorEmails.add(it)
+        val pattern = Pattern.compile("^[a-zA-Z0-9_+&*-]+(?:\\.[a-zA-Z0-9_+&*-]+)*@(?:[a-zA-Z0-9-]+\\.)+[a-zA-Z]{2,7}$")
```

Comment:
I would simplfy regex to only check if there is text before and after @ sign.
```suggestion
val regex = "^(.+)@(.+)$"
```

Reply:
Almost! 😄 
With this, the next value would be allowed: 
`matij@@@@@pev`
You can test it [here](https://regexr.com/5e6mp).

I propose something like this: `^[^@]+@{1}[^@]+$`,
which can be tested [here](https://regexr.com/5e6n5).

E-mail validation is a complex thing, but we don't need to be 99% sure that the user entered the correct e-mail address. So this approach with `something@something` is more than enough, in my opinion.